guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -18,11 +18,72 @@ You can list the available ece-tools commands using:
+
+    -  JIRA--MAGECLOUD-1506-->**[CLEAN_STATIC_FILES deploy variable]({{ page.baseurl }}/cloud/env/variables-deploy.html#clean_static_files)**—Added the capability to clean static content files generated during the build phase and updated the default to clean static content by default (``CLEAN_STATIC_FILES=`true``). Previously, you could only clean static content files generated during the deploy phase.
+
+-  **Logging**-Made the following changes to improve log messages and reduce log size:
+
+   -  JIRA--MAGECLOUD-2489-->Deployment failure log entries now include the command output from the operations that cause the failures.
+
+   -  JIRA--MAGECLOUD-2209-->Added logging for deployment failures that occur when generated factories required by some extensions cannot be generated correctly because the file system is in a read-only state.
+
+   -  JIRA--MAGECLOUD-2402-->Improved the deploy log to reduce log size and fix formatting issues caused by setup commands that used the interactive progress bar.
+
+   -  JIRA--MAGECLOUD-2227-->Eliminated unnecessary verbosity and updated the priority levels for some log statements.
+
+
+#### Resolved Issues
+
+-  JIRA--MAGECLOUD-2427-->Changed the default cron job configuration settings for history lifetime from 3d (4320 min) to 1h (60 min) to prevent performance issues and deployment failures that can occur when the cron queue fills too quickly.
+
+- JIRA-- MAGECLOUD-2182-->Fixed an issue with the [static content compression process]({{ site.baseurl }}cloud/env/variables-deploy.html#scd_compression_level) (`gzip`) that caused `not overwritten` and `no such file or directory` errors when referencing the compressed file during the deployment process.
+
+- JIRA--MAGECLOUD-2444-->Fixed an issue that prevented the `php ./vendor/bin/ece-tools config:dump` command from removing redundant sections from the `config.php` file during the dump process if the store locale is not specified.
+
+-  JIRA--MAGECLOUD-2520-->Fixed an issue with the `_merge` option for the [`SEARCH_CONFIGURATION`]({{ page.baseurl }}/cloud/env/variables-deploy.html#search_configuration) variable that caused incorrect merge results if the updated `.magento.env.yaml` configuration file did not include the `engine` parameter value along with other updated values. Now, the merge operation correctly overwrites only the parameter values included in the updated `.magento.env.yaml` without requiring you to specify the `engine` parameter value.
+
+-  JIRA-- MAGECLOUD-2515-->Fixed a Redis session locking issue that caused an Admin login delay. The fix is available for all versions of Magento v2.1 and v2.2.
```

Comment:
I suppose not for all in 2.1.x. Only started from 2.1.4

Reply:
Please also change 2515 description to: Fixed an issue with session locking being enabled for latest core versions of Magento by error due to inverted behavior in colinmollenhour/php-redis-session-abstract for disable_locking parameter starting v1.3.4

Reply:
@YPyltiai   -- this is a little difficult to understand. I proposed an revision on [MAGECLOUD-2515]([MAGECLOUD-2445](https://magento2.atlassian.net/browse/MAGECLOUD-2515)